src/css-select-browser-adapter-master/.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Auto detect text files and perform LF normalization
```

Comment:
Don't include the browser adapter in our repo. It is also available on npm:
https://www.npmjs.com/package/css-select-browser-adapter